.golangci.example.yml

```diff
@@ -236,8 +236,36 @@ linters-settings:
         # whether to check test functions (default true)
         skipTestFuncs: true
       ruleguard:
-        # path to a gorules file for the ruleguard checker
-        rules: ''
+        # Enable debug to identify which 'Where' condition was rejected.
+        # The value of the parameter is the name of a function in a ruleguard file.
+        #
+        # When a rule is evaluated:
+        # If:
+        #   The Match() clause is accepted; and
+        #   One of the conditions in the Where() clause is rejected,
+        # Then:
+        #   ruleguard prints the specific Where() condition that was rejected.
+        #
+        # The flag is passed to the ruleguard 'debug-group' argument.
+        debug: 'emptyDecl'
+        # Deprecated, use 'failOn' param.
+        # If set to true, identical to failOn='all', otherwise failOn=''
+        failOnError: false
```

Comment:
nit: maybe we should not mention deprecated parameters in the docs?

Reply:
It was deprecated because of golangci-lint https://github.com/golangci/golangci-lint/pull/2041#issuecomment-946601464 it seems fair to add that in our documentation.

Reply:
ok

Reply:
Let me create another PR to add that piece of information

Reply:
Never mind, it's already marked deprecated.